app/src/main/res/layout/activity_main.xml

```diff
@@ -1,12 +1,85 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.coordinatorlayout.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    tools:ignore="SpUsage, ContentDescription, RtlHardcoded"
     >
 
-    <include layout="@layout/lists_with_add_content"/>
+    <LinearLayout
+        android:id="@+id/focal_point"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
+        >
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginRight="8dp"
+            >
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/add_item"
+                android:layout_gravity="end|center_vertical"
+                android:layout_width="50dp"
+                android:layout_height="match_parent"
+                card_view:contentPadding="5dp"
+                card_view:cardUseCompatPadding="true"
+                card_view:cardBackgroundColor="@color/app_blue"
+                card_view:cardCornerRadius="3dp"
+                card_view:cardElevation="2dp"
+                >
+
+                <ImageView
+                    android:id="@+id/plus_icon"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    />
+            </androidx.cardview.widget.CardView>
+
+            <AutoCompleteTextView
```

Comment:
Looks like you copy-pasted from the list editing layout, which is a good start (as I mentioned before, it's very important to keep the code consistent), but we don't need this AutoCompleteTextView since it's for name suggestion on the list editing page (in case you want to enter in multiple copies of the same name for a raffle or something).